Wikipedia/Code/TalkPageCoffeeRollView.swift

```diff
@@ -0,0 +1,76 @@
+import UIKit
+
+final class TalkPageCoffeeRollView: SetupView {
+
+    // MARK: - Properties
+
+    var theme: Theme!
```

Comment:
I think you can remove the `!`s here if you're setting them in the `init`.

Reply:
Good catch, sorry – a holdover from before I switched my approach to requiring them at `init`. Updated!